ansible/roles/csm.ims-passthrough/README.md

```diff
@@ -0,0 +1,55 @@
+csm.ims-passthrough
+=========
+
+IMS exposes several runtime and image customization job setup parameters to ansible through environment variables. These
```

Comment:
The 'DKMS_ENABLED' is really the one that people want right now. That lets them know they can use dkms to modify kernel modules as part of the image customization playbook. Might want to mention that in the docs.